tests/tasks/assert_profile_present.yml

```diff
@@ -1,12 +1,17 @@
 # SPDX-License-Identifier: BSD-3-Clause
 ---
 - include_tasks: get_profile_stat.yml
-- name: "assert that profile '{{ profile }}' is present"
+- name: assert that profile {{ profile }} is present
```

Comment:
```suggestion
- name: assert that profile is present {{ profile }}
```

Reply:
Sure?
Is `assert that profile is present profile_name` better than `assert that profile profile_name is present`?

Reply:
> Sure? Is `assert that profile is present profile_name` better than `assert that profile profile_name is present`?

not sure if it is better, but ansible-lint 6.x does not like it when the `{{ template }}` is not at the end of `name` - see https://github.com/linux-system-roles/network/pull/580

Reply:
Wow... All right. Thanks.